iothub/service/src/DigitalTwin/Authentication/IotServiceClientCredentials.cs

```diff
@@ -33,6 +33,6 @@ public override Task ProcessHttpRequestAsync(HttpRequestMessage request, Cancell
         /// Return a SAS token
```

Comment:
Can you update the header comments to reflect that it isn't just a SAS token anymore?